src/components/Picker/pickerStyles/index.android.js

```diff
@@ -1,12 +0,0 @@
-// eslint-disable-next-line rulesdir/display-name-property
-import styles from '../../../styles/styles';
-
-const pickerStyles = isDisabled => ({
-    ...styles.picker(isDisabled),
-    inputAndroid: {
-        ...styles.picker(isDisabled).inputNative,
-        paddingLeft: 12,
-    },
-});
```

Comment:
@parasharrajat, looking at the diff, this is the only thing that looks like it could be causing this issue. What do you think?

Reply:
But this is a picker, not a text input, so I don't think it would even effect the Password input.